File: python/bc1012.py
from math import pow

pi = 3.14159

cVal = input().split(' ')

fVal = float(cVal[0])
sVal = float(cVal[1])
tVal = float(cVal[2])

aTri = (fVal * tVal) / 2
aCi = pi * pow(tVal,2)
aTra = ((fVal + sVal) * tVal) / 2
aQ = pow(sVal,2)
aR = fVal*sVal

print(f"TRIANGULO: {aTri:.3f}\nCIRCULO: {aCi:.3f}\nTRAPEZIO: {aTra:.3f}\nQUADRADO: {aQ:.3f}\nRETANGULO: {aR:.3f}")

File: cpp/bc1012.cpp
#include <iostream>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{
	double fVal, sVal, tVal, aTri, aCi, aTra, aQua, aRe;
	
	cin >> fVal >> sVal >> tVal;
	
	aTri = (fVal * tVal) / 2;
	aCi = pow(tVal,2) * 3.14159;
	aTra = ((fVal + sVal)* tVal) / 2;
	aQua = pow(sVal, 2);
	aRe = fVal * sVal;
	
	cout << "TRIANGULO: " << fixed << setprecision(3) << aTri <<  endl;
	cout << "CIRCULO: " << fixed << setprecision(3) << aCi << endl;
	cout << "TRAPEZIO: " << fixed << setprecision(3) << aTra << endl;
	cout << "QUADRADO: " << fixed << setprecision(3) << aQua << endl;
	cout << "RETANGULO: " << fixed << setprecision(3) << aRe << endl;
}
